src/Options.cs

```diff
@@ -7,12 +7,14 @@ namespace OpenInGVim
 {
     public class Options : DialogPage
     {
-        const string DefaultPathToExe = @"C:\Program Files (x86)\vim\vim80\gvim.exe";
+        const string DefaultPathToExe = @"C:\Program Files (x86)\vim\vim81\gvim.exe";
+        const string DefaultArguments = @"--remote-silent";
```

Comment:
If we open the windows into a new tab instead of existing one, the users will be able to get explicit visual feedback that the file has been opened.

```suggestion
        const string DefaultArguments = @"--remote-tab-silent";
```

Reply:
Could do.  Personally I tend to only use tabs almost like desktop views in vim to represent different views on files that I'm working on.  Then I tend to use buffers to open different files that I want to see.  However, this is your project and people use vim in different ways for sure :-)  Happy to make the change if you still want to.  At least now the setting dialog option works, I can override it on my own instance anyway :-)

Reply:
It is an open-source project. I am open to ideas obviously. In this particular case, the only reason why I think it's good to open the page in a new tab is the visual change on vim window you can see which makes it more apparent. And you're right, if someone wants to change it, they can do so in the settings.